src/test/resources/bandits/rac-experiments-bandits-beta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "flags": {
+    "test_bandit_1": {
+      "bandit": true,
+      "enabled": true,
+      "subjectShards": 10000
+    },
+    "test_bandit_2": {
+      "bandit": true,
+      "enabled": false,
+      "subjectShards": 10000
+    }
+  }
+}
```

Comment:
Did you decide against bandits being at the allocation level? I was thinking this would look like

```
{
  "flags": {
    "test_bandit_1": {
      "enabled": true,
      "subjectShards": 10000
      "allocations": {
        "allocation-1": {
          "bandit": true,
          "variations": [...]
        },
        "allocation-2": {
          "bandit": true,
          "variations": [...]
        }
      }
    },
    "test_bandit_2": {
      "enabled": false,
      "subjectShards": 10000
      "allocations": {
        "allocation-3": {
          "bandit": true,
          "variations": [...]
        },
        "allocation-4": {
          "bandit": true,
          "variations": [...]
        }
      }
    }
  }
}
```

I could imagine customers appreciating being able to run bandits with different targeting rules (e.g. one bandit for iphone users and a different bandit for android users) or overriding a bandit with a feature gate allocation

Reply:
You are a couple steps ahead of us! We do indeed one targeting rules and as of now have landed on a bandit being a type of "dynamic variant" (within an allocation).